app-emulation/euae/euae-0.8.29~wip4.recipe

```diff
@@ -74,7 +76,7 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	runConfigure ./configure --with-sdl --with-sdl-gfx --with-sdl-sound --with-sdl-gl \
+	runConfigure ./configure --with-sdl --with-sdl-gfx --with-sdl-sound \
```

Comment:
Shouldn't these options be picked up by pkg-config? (--with-sdl-*)
Also I see sdl-gfx in here but it's not in the requirements? (lib:libsdl_gfx) (ps, not checked yet)

Reply:
These are indeed needed, without them configure make use of "native BeOS" instead of SDL, but adding sdl_gfx shouldn't hurt (although I don't see any configure check for either sdl-sound nog sdl-gfx, config.log doesn't show anything either) :)